bpe/apply.py

```diff
@@ -1,145 +1,27 @@
+"""
+This is an old location of bpe jobs kept for backwards compatibility, for new setups using the subword-nmt based BPE,
+please use i6_core.label.bpe, for other setups please switch to the sentencepiece implementation
+"""
 __all__ = ["ApplyBPEModelToLexiconJob", "ApplyBPEToTextJob"]
 
-import gzip
-import subprocess as sp
-import os
-import sys
-import xml.etree.ElementTree as ET
 
-from sisyphus import *
+from i6_core.label.subword_nmt.apply import (
+    ApplyBPEModelToLexiconJob as _ApplyBPEModelToLexiconJob,
+)
+from i6_core.label.subword_nmt.apply import ApplyBPEToTextJob as _ApplyBPEToTextJob
```

Comment:
do we really need this
```
from i6_core.label.subword_nmt.apply import ApplyBPEToTextJob as _ApplyBPEToTextJob
class ApplyBPEToTextJob(_ApplyBPEToTextJob):
    pass
````
or would
```
from i6_core.label.subword_nmt.apply import ApplyBPEToTextJob
```
also be sufficient?

would there be implications for hashing (probably no as you tested this), comparison/isinstance test (probably no because of inheritance)?

Reply:
Just importing should be fine... I just did this out of habit when moving/renaming jobs. I can test this.

Reply:
Via: https://github.com/rwth-i6/i6_core/issues/255, https://github.com/rwth-i6/i6_core/pull/256

To add here: Just an import (alias) is not enough. It must be an own instance. It does not have the same hash because the hash depends on the full qual name of the class (which means including the package and module of the class where it is defined).